src/dev/flang/tools/fzjava/FZJava.java

```diff
@@ -426,6 +502,11 @@ void createOuter(String jfn)
     if (!_pkgs.contains(pkg))
       {
         _pkgs.add(pkg);
+        if (_existingFeatures.get(pkg.replace("/", ".")) != null)
+          {
+            // do not generate duplicate features
+            return;
+          }
         FeatureWriter.write(this, pkg, "_pkg", FeatureWriter.mangle(pkg.replace("/",".")) + " is\n");
```

Comment:
Please do not use `return` (or `break`) in nested blocks, this results in non-intuitive control flow. Better do
```
        if (_existingFeatures.get(pkg.replace("/", ".")) == null)
          {
            FeatureWriter.write(this, pkg, "_pkg", FeatureWriter.mangle(pkg.replace("/",".")) + " is\n");
          }
```